crates/sui-framework/packages/move-stdlib/sources/vector.move

```diff
@@ -25,42 +24,42 @@ module std::vector {
 
     #[bytecode_instruction]
     /// Create an empty vector.
-    native public fun empty<Element>(): vector<Element>;
+    public native fun empty<Element>(): vector<Element>;
```

Comment:
Sorry, those are side-effects of formatting. I hope no one minds.

Reply:
Consistency above all!